libs/types/src/election.ts

```diff
@@ -410,6 +410,16 @@ export const PrecinctsSchema = z
     }
   });
 
+// Represents a bubble option that should be displayed for selection on a ballot.
+export interface DisplayCandidate {
+  id: CandidateId;
+  // TODO(CROSS_ENDORSE): add more partyIds as needed.
+}
```

Comment:
Naming nit: `DisplayCandidate` kind of makes me think this should have the info needed to display a candidate. I was thinking of this more as a type that just represents a candidate in an ordering, so maybe `OrderedCandidate` or `CandidateOption` would make more sense. Then the ballot style field could be `orderedCandidatesByContest` or `orderedCandidateOptionsByContest`. I don't feel strongly about this at all though